### PR TITLE
Prototype

### PR DESCRIPTION
**Pull request by patmac510**
Hope you don't mind taking a look, it is a bit messy just warning you. And theres a couple functionality missing xD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 # comments-profile
```
Comment (mgalfano) [inline README.md]: Yay readme!!

Comment (mgalfano) [reply README.md]: Wait I just ran your component locally, you should edit this to say to run "npm run seed" instead

File: client/src/components/replyEntry.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      time: null,
+    };
+    this.getTime = this.getTime.bind(this);
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => this.getTime(), 1000);
+  }
+
+  getTime() {
```
Comment (mgalfano) [inline client/src/components/replyEntry.jsx]: replyEntry and commentsEntry (btw you should rename the files so they're capitalized) each have its own getTime method that are fairly expensive... consider moving the method up to App instead and pass the method down to each class to reduce the redundancy :)

File: public/index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Comment Section Module</title>
```
Comment (mgalfano) [inline public/index.html]: Yay you changed this! I'm running the older commit locally and this was going to be my first suggestion 👍

File: client/src/components/replyEntry.jsx (hunk 2 above)
Comment (patmac510) [reply client/src/components/replyEntry.jsx]: okay

**Pull request merged**